### PR TITLE
Personal MD file - Prahar Patel

### PR DESCRIPTION
**Pull request by praharpatel**
Created md file with all the details

### CODE REVIEW

File: pages/praharpatel.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Prahar Patel
+
+Hello! I am from Los Angeles, CA. I am a junior at California State Polytechnic University, Pomona pursuing Bachelor's in Computer Science. 
+<li>   I am using Windows 10 with AMD A6-6310 APU and AMD Radeon R4 graphics. Clock speed @ 1.8 GHz and a 64 bit system.
```
Comment (mappuji) [inline pages/praharpatel.md]: I see a ```<li>``` tag without a closing tag.

Here is quick cheat sheet for markdown [click here!](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)

Comment (dogi) [inline pages/praharpatel.md]: @praharpatel markdown please

examples of other personal pages are in http://open-learning-exchange.github.io/#!pages/team.md

**Comment by praharpatel** [comment]
thank you for the feedback. Changes are made. Please check :)

File: pages/praharpatel.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Prahar Patel
+
+Hello! I am from Los Angeles, CA. I am a junior at **California State Polytechnic University**, Pomona pursuing Bachelor's in Computer Science. 
+<li> I am using Windows 10 with AMD A6-6310 APU and AMD Radeon R4 graphics. Clock speed @ 1.8 GHz and a 64 bit system.
```
Comment (waryhermit) [inline pages/praharpatel.md]: You need to use three different kinds of markdown elements - no html please 👮

**Comment by praharpatel** [comment]
Done :)

**Comment by praharpatel** [comment]
Please ignore the above pull request for pages/firststeps.md. It was my mistake. If the personal md file is good, I request you to approve the changes

**Comment by waryhermit** [comment]
@praharpatel You will have to revert the changes - we will only merge if the only file changed is your markdown file 😢

File: pages/firststeps.md [hunk 3]
```diff
@@ -61,9 +61,9 @@ Follow the directions at [Git Repositories](gitandrepositories.md) to keep your
 
 Complete the tutorial under the [GitHub Issues tab](githubissues.md) and make sure to keep us posted in the Gitter chat. You should link in Gitter whenever you create an issue or when you comment on someone else's issue. Post as many issues as you can for improving the page as well as personal practice. If you know how to solve an issue, be sure to provide a detailed account of your research and show how to fix it. Next, fix the issue on your repo and create pull requests. Don't forget you can also comment and work on issues that you didn't create. The more you work and commit and push, the more impressive your GitHub profile will become. This is very important to potential employers, thus we encourage you to spend time to increase your numbers to strengthen your profile.
 
-## Step 7 - Nation BeLL:
+## Step 7 - Nation BeLLs:
```
Comment (dogi) [inline pages/firststeps.md]: @praharpatel please remove this file - cause it has nothing to do with your personal page

other question @praharpatel are you familiar with git branches?

File: pages/praharpatel.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Prahar Patel
```
Comment (dogi) [inline pages/praharpatel.md]: @praharpatel good start - please add more markdown elements

e.g. http://open-learning-exchange.github.io/#!pages/team.md

**Comment by zeivhann** [comment]
Here is a great succinct explanation for updating a pull request @praharpatel 
http://stackoverflow.com/questions/9790448/how-to-update-a-pull-request

Hopefully this helps!

**Comment by praharpatel** [comment]
Resolved the conflict. Added 3 markdown elements to personal file. Let me know if there is anything else needed. @dogi @waryhermit @mappuji

**Comment by waryhermit** [comment]
@praharpatel You still have 2 files changed.

**Pull request closed without merging**